cabby/rvs/templates.py

```diff
@@ -307,12 +317,13 @@ def add_features_to_template(template: Text, entity: geo_item.GeoEntity) -> Tupl
   template = template.replace(' a e', ' an e')
   template = template.replace(' a u', ' an u')
   template = template.replace(' a o', ' an o')
```

Comment:
This looks like old code -- are you sure you've merged this branch with the master branch?

Reply:
Now all files are merged